sickbeard/tvcache.py

```diff
@@ -400,9 +400,9 @@ def searchCache(self, episode, forced_search=False, downCurQuality=False):
 
     def listPropers(self, date=None):
         cache_db_con = self._get_db()
-        sql = b"SELECT * FROM [{provider_id}] WHERE name LIKE '%.PROPER.%' OR name LIKE '%.REPACK.%'".format(provider_id=self.provider_id)
+        sql = b"SELECT * FROM [{provider_id}] WHERE (name LIKE '%.PROPER.%' OR name LIKE '%.REPACK.%' OR name LIKE '%.REAL.%')".format(provider_id=self.provider_id)
```

Comment:
Do we still need this? What is the point of this?
Why should there be proper items in cache that were NOT already snatched?